.github/workflows/manubot.yaml

```diff
@@ -93,7 +93,7 @@ jobs:
       - name: Build Manuscript
         run: bash build/build.sh
       - name: Upload Artifacts
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
```

Comment:
https://github.com/actions/upload-artifact/releases/tag/v4.0.0
https://github.blog/changelog/2023-12-14-github-actions-artifacts-v4-is-now-generally-available/

Reply:
Here is the [log output](https://github.com/manubot/rootstock/actions/runs/10185388467/job/28174854640#step:9:18) from the upload artifact step

```
With the provided path, there will be 9 files uploaded
Artifact name is valid!
Root directory input is valid!
Beginning upload of artifact content to blob storage
Uploaded bytes 215686
Finished uploading artifact content to blob storage!
SHA256 hash of uploaded artifact zip is 470397e4848ba4fd526b226d5b9879e1553ae8e665f5dbb1b8e9b357e5375d94
Finalizing artifact upload
Artifact manuscript-10[18](https://github.com/manubot/rootstock/actions/runs/10185388467/job/28174854640#step:9:19)5388467-08f8521.zip successfully finalized. Artifact ID 1761448987
Artifact manuscript-10185388467-08f8521 has been successfully uploaded! Final size is [21](https://github.com/manubot/rootstock/actions/runs/10185388467/job/28174854640#step:9:22)5686 bytes. Artifact ID is 1761448987
Artifact download URL: https://github.com/manubot/rootstock/actions/runs/10185388467/artifacts/1761448987
```